client/pages/results.jsx

```diff
@@ -102,30 +127,22 @@ export default class Results extends React.Component {
             <Card className='orange-border padding-zero font-size-20 grey-background'>
               <Card.Header className='mx-0 font-titillium-web font-bold'>Last 3 Transactions</Card.Header>
               <ul className='px-4 py-2'>
-                <li>
-                  <Card.Title>Transaction ID: {this.state.transactionData[0].txid}</Card.Title>
-                  <ul>
-                    <li>
-                      <Card.Title>Block Height: {this.state.transactionData[0].status.block_height}</Card.Title>
-                    </li>
-                  </ul>
-                </li>
-                <li>
-                  <Card.Title>Transaction ID: {this.state.transactionData[1].txid}</Card.Title>
-                  <ul>
-                    <li>
-                      <Card.Title>Block Height: {this.state.transactionData[1].status.block_height}</Card.Title>
-                    </li>
-                  </ul>
-                </li>
-                <li>
-                  <Card.Title>Transaction ID: {this.state.transactionData[2].txid}</Card.Title>
-                  <ul>
-                    <li>
-                      <Card.Title>Block Height: {this.state.transactionData[2].status.block_height}</Card.Title>
-                    </li>
-                  </ul>
-                </li>
+                {this.state.transactionData.map((txData, i) => {
+                  if (i < 3) {
+                    return ([
+                      <li key={i}>
+                        <Card.Title>Transaction ID: {txData.txid}</Card.Title>
+                        <ul>
+                          <li>
+                            <Card.Title>Block Height: {txData.status.block_height}</Card.Title>
+                          </li>
+                        </ul>
+                      </li>
+                    ]);
+                  } else {
+                    return ([]);
+                  }
```

Comment:
Instead of using a conditional, just slice the array. Also, there's no reason to wrap every returned element in an array.

```js
this.state.transactionData.slice(0, 3).map((transactionData, index) => {
```